Packs/HashiCorp-Vault/Integrations/HashiCorpVault/HashiCorpVault.yml

```diff
@@ -234,66 +234,74 @@ script:
     arguments: []
     outputs:
     - contextPath: HashiCorp.Policy.Name
-      description: Policy name
+      description: Policy name.
       type: string
     description: Lists all configured policies
   - name: hashicorp-get-policy
     arguments:
     - name: name
       required: true
-      description: Policy name
+      description: Policy name.
     outputs:
     - contextPath: HashiCorp.Policy.Name
-      description: Policy name
+      description: Policy name.
       type: string
     - contextPath: 'HashiCorp.Policy.Rule.Path '
-      description: Policy rule path
+      description: Policy rule path.
       type: string
     - contextPath: HashiCorp.Policy.Rule.Capabilities
-      description: Policy rule capabilities
+      description: Policy rule capabilities.
     description: Get information for a policy
   - name: hashicorp-seal-vault
     arguments: []
     description: If you suspect your data has been compromised, you can seal your
-      vault to prevent access to your secrets
+      vault to prevent access to your secrets.
     execution: true
   - name: hashicorp-unseal-vault
     arguments:
     - name: key
-      description: 'Single master key '
+      description: Single master key.
     - name: reset
       auto: PREDEFINED
       predefined:
       - "true"
-      description: Reset the unseal project
+      description: Reset the unseal project.
     description: Use a single master key share to unseal the vault. If the master
-      key shares threshold is met, vault will attempt to unseal the vault. Otherwise,
+      key shares threshold is met, the vault will attempt to unseal the vault. Otherwise,
```

Comment:
Is this correct....
the **vault** will attempt to unseal the vault?
Should it be 
the **key** will attempt to unseal the vault?